### PR TITLE
Added Footer Code and Added Assets folder

### PR DESCRIPTION
**Pull request by mayankrajendrat**

### CODE REVIEW

File: footer.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<footer>
```
Comment (Harshit77Nav) [inline footer.html]: It should be a proper html structure

**Review by Shreanshu — COMMENTED** [review]
Add proper html structure.

File: footer.html (hunk 1 above)
Comment (rjrajujha) [inline footer.html:1]: no correction needed

**Review by Shreanshu — COMMENTED** [review]
Include proper html structure.

**Review by Shreanshu — APPROVED** [review]

**Pull request merged**